edbee-lib/edbee/models/texteditorconfig.cpp

```diff
@@ -405,6 +405,23 @@ void TextEditorConfig::setShowWhitespaceMode(const QString& str)
     }
 }
 
+/// Sets whether autocomplete comes up automatically, or only manually(manual trigger isn't implemented yet)
+/// @see TextEditorConfig::autocompleteAutoShow
+void TextEditorConfig::setAutocompleteAutoShow(bool enable)
+{
+    qDebug() << "setAutocompleteAutoShow(" << enable << ")\n";
```

Comment:
Still need this or can you clean it up?

Reply:
Debug lines removed via last two commits. Should be good to go, probably a good idea to have someone test.